File: Assignment_11/Program2/program2.java
////////////////////////////////////////////////////////////////////////////////
// 
// File: program2.java
// Description: Used to accept a nuber from user and display below pattern.
// Input: 5
// Output: 5 # 4  # 3  # 2 # 1 #
// Author: Parimal Kelkar
// Date: 25/05/2025
// 
////////////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

class program2
{
    public static void main(String args[])
    {
        Scanner sObj = new Scanner(System.in);

        int iValue = 0;

        System.out.println("Enter number: ");
        iValue = sObj.nextInt();

        Pattern pObj = new Pattern();
        pObj.Display(iValue);
        
    }
}

class Pattern
{
    ////////////////////////////////////////////////////////////////////////////
    //
    // Method: Display()
    // Description: Accepts a number and displays the mentioned pattern.
    // Input: int
    // Output: void
    // Author: Parimal Kelkar
    // Date: 25/05/2025
    //
    ////////////////////////////////////////////////////////////////////////////
    void Display(int iNo)
    {
        int iCnt = 0;
        char ch = '\0';

        // Updator
        if(iNo < 0)
        {
            iNo = -iNo;
        }

        for(iCnt = iNo; iCnt >= 1; iCnt--)
        {
            System.out.print(iCnt + "\t#\t");
        }
    }
}

File: Assignment_40/program2/program2.c
////////////////////////////////////////////////////////////////////////////////
//
//  File name: Program2.c
//  Description: Used to print below pattern recursively.
//  Output: 1 2 3 4 5
//  Author: Parimal Kelkar
//  Date: 21/07/2025
//
////////////////////////////////////////////////////////////////////////////////

#include<stdio.h>

////////////////////////////////////////////////////////////////////////////////
//
//  Name: Display()
//  Description: Displays a pattern recursively.
//  Input: void
//  Output: void
//  Author: Parimal Kelkar
//  Date: 21/07/2025
//
////////////////////////////////////////////////////////////////////////////////

void Display()
{
    static int i = 1;

    if(i <= 5)
    {
        printf("%d\t", i);
        i++;
        Display();
    }
}

int main()
{
    Display();

    return 0;
}